spec/requests/posts_spec.rb

```diff
@@ -1,7 +1,44 @@
 require 'rails_helper'
 
 RSpec.describe 'Posts', type: :request do
-  describe 'GET /index' do
-    pending "add some examples (or delete) #{__FILE__}"
+  describe 'GET /users/:user_id/posts' do
+    let(:user) { User.create(name: 'Test User') }
+
+    before do
+      get "/users/#{user.id}/posts"
+    end
+
+    it 'returns a successful response' do
+      expect(response).to have_http_status(200)
+    end
+
+    it 'renders the index template' do
+      expect(response).to render_template(:index)
+    end
+
+    it 'includes the list of posts for the selected user in the response body' do
+      expect(response.body).to include('List of post for selected User')
+    end
+  end
+
+  describe 'GET /users/:user_id/posts/:id' do
```

Comment:
- [ ] Great job working on this milestone. You should test the index and `show` actions and that should be reflected in your `describe`. See below.
```
describe 'GET /show' do
  ...
end
```